doc/changes/latest.rst

```diff
@@ -21,3 +21,8 @@ Enhancements
 
 Changes
 -------
+
+- The documentation for :func:`~image.threshold_img` has been improved,
```

Comment:
In general we have the convention of one line per entry in this file (see [PR Structure](https://nilearn.github.io/stable/development.html#pr-structure)). I know this is broken sometimes and I don't have a strong opinion about it but better to try to keep things consistent.

Reply:
woopsy

I am trying to use [semantic linebreaks](https://sembr.org/) more and more, so I completely overlooked this.

To make it harder to miss, would it make sense to add links to the relevant headings in the [PR checklist](https://github.com/nilearn/nilearn/blob/b7e5efdd37a6b4cc276763fc5f2a2cde81f7af73/.github/workflows/auto-comment.yml#L22) ?

Reply:
Yes I think that's a good idea! Can you open an issue for that?